lib/content/getting-started.md

```diff
@@ -38,17 +38,17 @@ First things first, we need to setup a base pal project. Run the following:
 npx hpal new paldo-riddles
 cd paldo-riddles
 npm install
+
+# make your first commit to init project history
+git add --all
```

Comment:
I believe all the files are already added after starting a new project.  However, after installation there is one new file, package-lock.json, that this would add.

Reply:
I see now that this mirrors the boilerplate readme.  Maybe it is easier to leave like this than get granular about adding specific files 👍